bokehjs/test/unit/models/tools/inspectors/hover_tool.ts

```diff
@@ -74,6 +76,32 @@ describe("HoverTool", () => {
     })
   })
 
+  describe("line_policy", () => {
+    it("should support 'none' line_policy with mode=vline", async () => {
+      const tooltips: [string, string][] = [["x", "$x"], ["y", "$y"]]
+      const hover = new HoverTool({mode: "vline", line_policy: "none", tooltips})
+      const p = fig([200, 200], {toolbar_location: null, min_border: 0, x_range: [1, 3], y_range: [0, 2]})
```

Comment:
`fig()` already sets `toolbar_location: null`. `min_border` is unnecessary if the rest of the test is rewritten using `PlotActions`. Are ranges actually relevant to this test?